apps/src/signUpFlow/signUpFlowStyles.module.scss

```
@@ -0,0 +1,385 @@
+@import 'color', 'breakpoints', 'style-constants', 'typography';
+
+@mixin link-style {
+  color: $light_black;
+  text-decoration: underline;
+  font-weight: 500;
+}
+
+body {
+  background-color: $light_gray_50 !important;
+}
+
+main.wrapper {
+  padding-block: 1rem;
+}
+
+.contentContainer {
+  max-width: $content-width;
+  margin: auto;
+  font-size: 1rem;
+}
+
+input {
+  width: 100%;
+  box-sizing: border-box;
+  height: 46px !important;
+  text-indent: 0.25rem;
+}
+
+select {
+  padding-top: 11px !important;
+  padding-bottom: 11px !important;
+  height: 46px !important;
+  font-weight: 400 !important;
+}
+
+p {
+  margin-bottom: 0 !important;
+}
+
+.newSignupFlow {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding-block: 2rem;
+  margin-inline: auto;
+
+  .shortBottomMargin {
+    margin-bottom: .5rem !important;
+  }
+
+  .containerWrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background-color: $white;
+    padding: 30px;
+    gap: 16px;
+    border-radius: 4px;
+    border: 1px solid $light_gray;
+    margin: 2.5rem 0 1rem 0;
+  }
+
+  .dividerContainer {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: 0 20px;
+  }
+
+  .verticalDividerTop,
+  .verticalDividerBottom {
+    width: 1px;
+    background-color: #ccc;
```

Nit: replace this with the color variable:
```suggestion
    background-color: $light_gray_200;
```

---

Thanks for catching this!